core/src/main/kotlin/links/DRI.kt

```diff
@@ -95,7 +95,7 @@ data class TypeReference(val classNames: String, val typeBounds: List<TypeRefere
 
         private fun from(t: KotlinType): TypeReference =
             when (val d = t.constructor.declarationDescriptor) {
-                is TypeParameterDescriptor -> from(d)
+                is TypeParameterDescriptor -> TypeReference("todo:${d.name}")
```

Comment:
What is the purpose of this?

Reply:
Right now there's infinite recursion when used on bigger project like Kotlin stdlib. It's temporary "solution" to get it to finish.